src/BitcoinLib/BitcoinLib.csproj

```diff
@@ -7,6 +7,10 @@
     <Copyright>George Kimionis 2014 - 2017</Copyright>
     <Version>1.4.2.0</Version>
     <FileVersion>1.4.2.0</FileVersion>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Isn't this declaration a duplicate of [this](https://github.com/nixxholas/BitcoinLib/blob/bd1604fb84192793849bd659abdcbfd1df71322a/src/BitcoinLib/BitcoinLib.csproj#L4) one?

```xml
<TargetFrameworks>netstandard20;net461</TargetFrameworks>
```

Reply:
Apologies, you're right

Reply:
No worries! Would it be possible to remove this and force push to your branch so that the PR is updated?

Reply:
@nixxholas, if you can get rid of this duplicate definition and rebase this with `master` we can merge it. `netstandard20` is merged, so this one should now target `master` -- sorry for the inconvenience, we were having issues with Travis CI, plus we had to see how the NuGet Package goes now that we target both frameworks.

Reply:
@moodmosaic apologies, have been really busy for milestone. Is this still required to be done?

Reply:
Yes it is :/ That's because we've merged netstandard20 branch into master and released v1.6.0 with .NET Framework 4.6 and .NET Standard 2.0 support.